src/mongo_odm/mongo_odm_child_autogen.hpp

```diff
@@ -17,9 +17,16 @@
 
```

Comment:
I think we propbably don't want this file in version control? If it's auto-generated, is there a reason not to gitignore it?